### ISSUE TITLE
waitForViewWithAccessibilityLabel fails to scroll to the table view cell

### ISSUE THREAD

**Issue by paulz** [kif-framework/KIF#402]
Looks like the expected behavior was broken recently, see:
https://github.com/kif-framework/KIF/commit/19e81b19300a1e6745bd27d790467728a40cc631#commitcomment-6449515
